test/browser-plugins/plugins.test.js

```diff
@@ -181,79 +186,185 @@ describe('Plugins', () => {
+            });
 
-        test('should use persistent context by default', async () => {
-            const plugin = new PlaywrightPlugin(playwright.chromium);
-            browserController = plugin.createController();
+            test('should use persistent context by default', async () => {
+                const plugin = new PlaywrightPlugin(playwright[browserName]);
+                const browserController = plugin.createController();
 
-            const launchContext = plugin.createLaunchContext();
+                const launchContext = plugin.createLaunchContext();
 
-            const browser = await plugin.launch(launchContext);
-            browserController.assignBrowser(browser, launchContext);
-            browserController.activate();
+                browser = await plugin.launch(launchContext);
+                browserController.assignBrowser(browser, launchContext);
+                browserController.activate();
 
-            const page = await browserController.newPage();
-            const context = await page.context();
-            await browserController.newPage();
+                const page = await browserController.newPage();
+                const context = await page.context();
+                await browserController.newPage();
 
-            expect(context.pages()).toHaveLength(3);
-        });
+                expect(context.pages()).toHaveLength(3);
+            });
 
-        test('should pass launch options to browser', async () => {
-            const plugin = new PlaywrightPlugin(playwright.chromium);
+            test('should pass launch options to browser', async () => {
+                const plugin = new PlaywrightPlugin(playwright[browserName]);
 
-            jest.spyOn(plugin.library, 'launch');
-            const launchOptions = {
-                foo: 'bar',
-            };
-            const launchContext = plugin.createLaunchContext({ launchOptions, useIncognitoPages: true });
-            browserController = await plugin.launch(launchContext);
-            expect(plugin.library.launch).toHaveBeenCalledWith(launchOptions);
+                jest.spyOn(plugin.library, 'launch');
+                const launchOptions = {
+                    foo: 'bar',
+                };
+                const launchContext = plugin.createLaunchContext({ launchOptions, useIncognitoPages: true });
+                browser = await plugin.launch(launchContext);
+                expect(plugin.library.launch).toHaveBeenCalledWith(launchOptions);
+            });
+            describe('PlaywrighBrowser', () => {
+                test('should create new page', async () => {
+                    const plugin = new PlaywrightPlugin(playwright[browserName]);
+
+                    const launchContext = plugin.createLaunchContext();
+                    browser = await plugin.launch(launchContext);
+                    const page = await browser.newPage();
+
+                    expect(page.close).toBeDefined();
```

Comment:
I would be a bit stricter here. `close` is a very common property on a lot of objects. I'd check that it's a function. And also that `evaluate` is a function.